src/server/network/protocol/protocolgame.cpp

```diff
@@ -3290,12 +3290,6 @@ void ProtocolGame::sendCreatureOutfit(const std::shared_ptr<Creature> &creature,
 	msg.add<uint32_t>(creature->getID());
 	AddOutfit(msg, newOutfit);
 
-	if (!oldProtocol && newOutfit.lookMount != 0) {
-		msg.addByte(newOutfit.lookMountHead);
-		msg.addByte(newOutfit.lookMountBody);
-		msg.addByte(newOutfit.lookMountLegs);
-		msg.addByte(newOutfit.lookMountFeet);
-	}
 	writeToOutputBuffer(msg);
```

Comment:
load this (sendCreatureOutfit and AddCreature) into AddOutfit
 (same packets)
![image](https://github.com/user-attachments/assets/1f96b17a-0a03-404e-bd24-2597f2a6e5e0)